src/test/java/de/uni_passau/fim/se2/pipeline_helper/checkers/LineLengthCheckerTest.java

```diff
@@ -29,65 +28,57 @@ static void init() throws URISyntaxException {
 
     @Test
     void checkFailedSingleViolation() throws Exception {
+        Path invalidDir = dir.resolve("invalid/");
         final LineLengthChecker checker = new LineLengthChecker(
-            dir, FilteredFilesStream.files(dir, "java"), 80
+            invalidDir, FilteredFilesStream.files(invalidDir, "java"), 80
         );
         final CheckerResult result = checker.check();
         assertThat(result.getName()).contains("LineLengthChecker");
         assertThat(result.isSuccessful()).isFalse();
         assertThat(result.getMessage()).contains(
             withNormalisedNewline(
                 """
-                    invalid%sInvalidFileSingleViolation.java, on 1 line:
-                        -> line 7, length 82""".formatted(File.separator)
+                    InvalidFileSingleViolation.java, on 1 line:
+                        -> line 7, length 82"""
             )
         );
     }
 
     @Test
-    void checkFailedTwoViolationsDifferentFiles() throws Exception {
+    void checkFailedDifferentFiles() throws Exception {
+        Path invalidDir = dir.resolve("invalid/");
         final LineLengthChecker checker = new LineLengthChecker(
-            dir, FilteredFilesStream.files(dir, "java"), 80
+            invalidDir, FilteredFilesStream.files(invalidDir, "java"), 80
         );
         final CheckerResult result = checker.check();
         assertThat(result.getName()).contains("LineLengthChecker");
         assertThat(result.isSuccessful()).isFalse();
         assertThat(result.getMessage()).contains(
             withNormalisedNewline(
                 """
-                    invalid%sInvalidFile.java, on 2 lines:
+                    InvalidFile.java, on 2 lines:
                         -> line 7, length 82
-                        -> line 10, length 99""".formatted(File.separator)
+                        -> line 10, length 99"""
             )
         );
-        assertThat(result.getMessage()).contains(
-            withNormalisedNewline(
-                """
-                    also_invalid%sInvalidFile.java, on 2 lines:
-                        -> line 7, length 82
-                        -> line 10, length 99""".formatted(File.separator)
-            )
+        withNormalisedNewline(
+            """
+                invalidFileSingleViolation.java, on 1 line:
+                    -> line 7, length 82"""
         );
     }
 
     @Test
-    void checkFailedMoreThanTwoViolations() throws Exception {
```

Comment:
Why were the two previous fixtures and corresponding tests removed?

Reply:
I didn't think they added anything. 2 vs. 4 violations doesn't make a difference, we could also test 7 or 11 but I don't think that's necessary. And the violations in two different files are also tested elsewhere.

> Removed some tests and related files that I personally found unnecessary (that were also added by me a while ago)

Reply:
You could argue that redundancy isn't too bad when it comes to testing. We could also leave them in if you think that's better.

Reply:
Okay to remove then, I apparently skipped over that par of the PR description.